File: Week-02/Day-04/draw-triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

number = int(input("Milyen magas legyen a piramisod? "))
height = ""

for i in range(0,number):
    height += "*"
    print(height)

File: Week-07/Day-02/draw-triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

var j = ''

for(var i = 0; i < lineCount; i++){
  j += '*';
  console.log(j);
}
